README.md

```diff
@@ -404,6 +404,22 @@ you can use `CHAMBER_AWS_REGION` to override just for chamber.
 If you'd like to use a custom SSM endpoint for chamber, you can use `CHAMBER_AWS_SSM_ENDPOINT`
 to override AWS default URL.
 
+## AWS Secrets Manager
+Chamber supports AWS Secrets Manager as an optional backend. For example:
+
+```
+chamber -b secretsmanager write myservice foo fah
+chamber -b secretsmanager write myservice foo2 fah2
+```
+
+This will result in one secret being generated with the following JSON value:
```

Comment:
It'd be better for the README not to commit to / advertise the particular JSON format that the secrets manager backend uses. Based on comments in the original PR and the general sentiment I get from the code, there's no guarantee of interoperability in how chamber stores things, so we'd want to keep the freedom to revamp the format used for Secrets Manager in the future. (Plus we'd have to keep updating this doc!)

If dropping this makes the new section feel too small, it could be replaced with anything a chamber user needs to configure in Secrets Manager for chamber to work with it.